### ISSUE TITLE
Version 6.4.2 hangs

### ISSUE THREAD

**Issue by sandman202** [a85/POSTMan-Chrome-Extension#5345]
Sending GET request with one parameter to pull user information.
It hangs on "Loading..." It does not timeout. There are no errors in log file.
Using v6.4.2 Windows. 

It works properly in the browser. Also works on older v5.5.3 Chrome app.

**Comment by sujayvenaik** [comment]
Hey @sandman202 

Can you go to View -> Developer -> Show Dev Tools (Current View) and send us the logs from that console.

Also, can you generate the cURL command for the request and test if you are able to receive the response for it?
You can get cURL command for the request by using the `Code` feature which is present below the send/save button.

**Comment by sandman202** [comment]
Hey sujayvenaik!
Thanks for the quick reply. After I created this issue, I figured out the cause through reading a post on disabling Syncing.

On the 6.4.2 version, I was logged in, which tries to sync. The post said 6.x version no longer have the ability to disable syncing. Instead, it said to log out. This will prevent syncing. The company I work for, doesn't want anything stored in the cloud. By logging out, I was able to execute the REST calls as expected. No more hanging.
